### DISCUSSION TITLE
upgrade not being done

### DISCUSSION THREAD

**Discussion by kdmiller45** [mastodon/mastodon#18550] [Q&A]
I am currently at 3.51 and am trying to upgrade to 3.53 and perform the following
as user mastodon
cd \home\mastodon\live
git fetch && git checkout v3.5.3
bundle install
yarn install
Non-Docker: RAILS_ENV=production bundle exec rails assets:precompile

as user root
Restart all Mastodon processes
systemctl daemon-reload
systemctl start mastodon-web mastodon-sidekiq mastodon-streaming

it still shows 3.51 I have even restarted the server
site still works just not at 3.53
https://topic-master.com

assistance would be nice

**Comment by ClearlyClaire** [comment]
> systemctl start mastodon-web mastodon-sidekiq mastodon-streaming

you want to `restart` instead of `start`

**Reply by kdmiller45** [reply]
made that mod, but even after restarting server still shows 3.51

Is there a log I can check

**Comment by kdmiller45** [comment]
Not sure what I did wrong maybe type git fetch && git checkout v3.53 but I redid it again  and now at 3.5.3

Thanks for the help
